osmosis-1/osmosis-1.chainlist.json

```diff
@@ -9082,6 +9082,65 @@
         "ibc-go",
         "ibc-transfer"
       ]
+    },
+    {
```

Comment:
Please do not modify this chainlist file directly.

Reply:
Got it, removed